chart/templates/_commonChecks.tpl

```diff
@@ -161,6 +216,57 @@ Return customer values to use in preflights and support-bundle
     value: {{ include "carto.postgresql.databaseName" . }}
   - name: WORKSPACE_POSTGRES_USER
     value: {{ include "carto.postgresql.user" . }}
+  - name: WORKSPACE_TENANT_ID
+    value: {{ .Values.cartoConfigValues.selfHostedTenantId | quote }}
+  {{- if not .Values.commonBackendServiceAccount.enableGCPWorkloadIdentity }}
+  - name: GOOGLE_APPLICATION_CREDENTIALS
+    value: {{ include "carto.google.secretMountAbsolutePath" . }}
+  {{- end }}
+  - name: WORKSPACE_THUMBNAILS_BUCKET
+    value: {{ .Values.appConfigValues.workspaceThumbnailsBucket | quote }}
+  - name: WORKSPACE_IMPORTS_BUCKET
+    value: {{ .Values.appConfigValues.workspaceImportsBucket | quote }}
+  - name: WORKSPACE_THUMBNAILS_PROVIDER
+    value: {{ .Values.appConfigValues.storageProvider | quote }}
+  - name: WORKSPACE_IMPORTS_PROVIDER
+    value: {{ .Values.appConfigValues.storageProvider | quote }}
+  - name: WORKSPACE_THUMBNAILS_PUBLIC
+    value: {{ .Values.appConfigValues.workspaceThumbnailsPublic | quote }}
```

Comment:
Is not better to just copy this? https://github.com/CartoDB/carto-selfhosted-helm/blob/e3334154072fc058597e0ff4096250f5db650c7e/manifests/kots-helm.yaml#L93

Reply:
Same as above ⬆️